src/utils/plotSpecs/generateEmbeddingContinuousSpec.js

```diff
@@ -2,7 +2,7 @@
 
 import { getAllCells, getSampleCells } from 'utils/cellSets';
 
-const generateSpec = (config, plotData) => {
+const generateSpec = (config, method, plotData) => {
```

Comment:
Why can't we use the config and need to pass a separate parameter now?

Reply:
Using this to generate the default axis names, instead of dispatching a redux action updateEmbeddingAxesIfNecessary every time, like it was done before.